fix permissions for disk encryption set

r0bnet opened the pull request:
The old permissions that were documented didn't work as expected. The ones I used in this PR are working

favoretti commented:
Hi @r0bnet, thank you for this PR.

When I look at https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/keyvault/key_vault_key_resource.go#L88, the validation there will not allow values you propose in docs.
These are the enum values that come from the SDK:

```
const (
	// Decrypt ...
	Decrypt JSONWebKeyOperation = "decrypt"
	// Encrypt ...
	Encrypt JSONWebKeyOperation = "encrypt"
	// Sign ...
	Sign JSONWebKeyOperation = "sign"
	// UnwrapKey ...
	UnwrapKey JSONWebKeyOperation = "unwrapKey"
	// Verify ...
	Verify JSONWebKeyOperation = "verify"
	// WrapKey ...
	WrapKey JSONWebKeyOperation = "wrapKey"
)
```

Can you please elaborate on how you managed to get the values you propose past resource validation? :)

r0bnet commented:
Hey @favoretti,

You're right. I changed the wrong thing. I wanted to adjust the access_policy for the DES managed identity on the KV and fixed that with another commit. Sorry for that. :)

favoretti reviewed (approved):
Thank you @r0bnet, LGTM 👍🏻
